src/date-picker/README.md

```diff
@@ -0,0 +1,33 @@
+# igx-date-picker
+
+The **igx-date-picker** component allows you to choose date from calendar
+which is presented into input field.
+
+# Usage
+```html
+<igx-date-picker [formatter]="curtomFormatter" [dateValue]="dateValue">
+</igx-date-picker>
+```
+
+# API Summary
+| Name   |      Type      |  Description |
+|:----------|:-------------:|:------|
+| `formatter` |  function | Sets the date format. |
+| `dateValue` | Date | Sets the initial date value. |
+| `onOpened` | event | Emitted when a datePicker calendar is being opened. |
```

Comment:
This is a custom event in the datePicker component, which is emitted when you click on the input and the calendar appears.
The implementation of this event, comes from the spec for the datePicker